assets/app.js

```diff
@@ -737,6 +770,16 @@ new Vue({
         }
     },
     methods: {
+        validateRejectedProduct: function (type, product) {
+            if(product['rejected_'+type]){
+                if(product[type] == null || Object.keys(product[type]).length === 0 || product.length === 0){
```

Comment:
instead of doing if(...), I'd just `return product[type] == null || Object.keys(product[type]).length === 0 || product.length === 0`

thoughts?